src/containers/audio-rewards-page/store/mobileSagas.ts

```diff
@@ -0,0 +1,20 @@
+import { takeEvery, put } from 'redux-saga/effects'
+
+import { updateHCaptchaScore } from 'containers/audio-rewards-page/store/slice'
+import { MessageType } from 'services/native-mobile-interface/types'
+
+function* watchUpdateHCaptchaScore() {
+  yield takeEvery(MessageType.UPDATE_HCAPTCHA_SCORE, function* (action: {
```

Comment:
we should be able to do this without explicitly passing the message using the common client stuff right? @sliptype

Reply:
No this is correct, the store only syncs in one direction. Passing data from the mobile client to the client still needs to be done with message passing. When everything is implemented in RN there will no longer be a need for message passing

Reply:
👍  yes yes